mitmproxy/tools/console/statusbar.py

```diff
@@ -178,6 +178,8 @@ def get_status(self):
             r.append(("heading_key", "T"))
             r.append("CP:%d]" % len(self.master.options.tcp_hosts))
         if self.master.options.intercept:
+            if self.master.options.intercept_active:
+                r.append("I ")
```

Comment:
Minor suggestion: How about marking this with an X`within the parantheses when not active, so that we have a more clean `[]` status bar?